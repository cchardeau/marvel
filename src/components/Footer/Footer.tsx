import React from 'react'
import Typography from '@material-ui/core/Typography'
import { makeStyles } from '@material-ui/core/styles'

const useStyles = makeStyles(theme => ({
  footer: {
    backgroundColor: theme.palette.background.paper,
    padding: theme.spacing(6)
  }
}))

const Footer: React.FC = () => {
  // styling
  const classes = useStyles()

  return (
    <footer className={classes.footer}>
      <Typography variant="h6" align="center" gutterBottom>
        Corentin Chardeau
      </Typography>
      <Typography variant="subtitle1" align="center" color="textSecondary" component="p">
        Peaks @ 2019
      </Typography>
    </footer>
  )
}

export default Footer