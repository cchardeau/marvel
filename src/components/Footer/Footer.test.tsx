import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'

import Footer from './Footer'

it('renders without crashing', () => {
  const div = document.createElement('div')
  ReactDOM.render(<Footer />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it('renders correctly', () => {
  const output = renderer.create(<Footer />).toJSON()
  expect(output).toMatchSnapshot()
})
