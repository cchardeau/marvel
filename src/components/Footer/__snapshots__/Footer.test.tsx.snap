// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="makeStyles-footer-32"
>
  <h6
    className="MuiTypography-root MuiTypography-h6 MuiTypography-gutterBottom MuiTypography-alignCenter"
  >
    Corentin Chardeau
  </h6>
  <p
    className="MuiTypography-root MuiTypography-subtitle1 MuiTypography-colorTextSecondary MuiTypography-alignCenter"
  >
    Peaks @ 2019
  </p>
</footer>
`;
